data/src/main/java/com/example/data/common/AccessTokenCache.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.data.common
+
+import android.content.SharedPreferences
+import com.example.data.common.Constants.ACCESS_OAUTH_TOKEN
+import com.example.data.common.Constants.ACCESS_OAUTH_TOKEN_SECRET
+
+data class AccessTokenCache(
```

Comment:
Data classes shouldn't do any operations, only contain data.
This class might be transformed to some SharedPref wrapper to save and read values in prefs.
- make it not data class 
- make it singleton 
- set some more common class name like Preferences